deploy/lib/parca/parca.libsonnet

```diff
@@ -95,99 +98,6 @@ function(params) {
     },
   },
 
-  podSecurityPolicy: {
```

Comment:
whether we like it or not, there are inevitably going to be clusters around that are old enough and have PSPs enabled, can we move this to a `withPodSecurityPolicy` patch, that's not applied by default for now?

Reply:
They are optional now.